src/simulator/api.yaml

```diff
@@ -2124,15 +2124,20 @@ components:
       properties:
         isValid:
           type: boolean
-        authChannels:
-          type: array
-          items:
-            $ref: '#/components/schemas/ConsentRequestChannelType'
-        authUri:
-          type: string
-          description: >-
-            The callback uri that the pisp app redirects to for user to complete
-            their login.
+        data:

```

Comment:
`data` is very generic name, what about `validateResponse` ?

Reply:
I think data is fine, isValid is validation response and data contains some extra information returned by dfsp